### ISSUE TITLE
Users are not warned against autoplaying videos

### ISSUE THREAD

**Issue by karlgroves** [WordPress/gutenberg#15338]
Users are not warned against autoplaying videos

-   **Severity:** Low
-   **Affected Populations:**
    -   Blind
    -   Low-Vision
    -   Motor Impaired
    -   Cognitively Impaired
-   **Platform(s):**
    -   All / Universal
-   **Components affected:**
    -   Block Types
    -   Output Markup

#### Issue description

Authors who create Video or Audio blocks can, in the Block panel, set
the media to autoplay.

Autoplaying video can interfere with users of text-to-speech software
such as screen readers and screen magnifiers, as well as make focussing
on content difficult for users with cognitive disabilities.

#### Remediation Guidance

When users are adding video or audio content and select the "Autoplay"
checkbox in the Block panel, add a warning that informs users of some of
the accessibility issues with autoplay, similar to the color contrast
warnings shown when users choose low-contrast color combinations.

**Note:** This issue may be a duplicate with other existing accessibility-related bugs in this project. This issue comes from the Gutenberg accessibility audit, performed by [Tenon](https://www.tenon.io) and funded by [WP Campus](https://wpcampus.org/). This issue is GUT-114 in Tenon's report

**Comment by kjellr** [comment]
Thanks, @karlgroves! 

> When users are adding video or audio content and select the "Autoplay"
checkbox in the Block panel, add a warning that informs users of some of
the accessibility issues with autoplay, similar to the color contrast
warnings shown when users choose low-contrast color combinations.

There's an existing issue here for this solution (#13550), so I'm going to close this issue as a duplicate. If this should be re-opened for any reason, just leave a note here.

**Comment by realworldev** [comment]
@aduth Why is this issue added to **Done**? Please remove, related #13550 is not resolved yet.

**Comment by aduth** [comment]
@realworldev I agree it's not a great title to reflect the current status of the issue.

That said, the idea with the project board was to have a basic gauge for progress with minimal ongoing triage effort by leveraging [project automation](https://help.github.com/en/articles/configuring-automation-for-project-boards). Any issue which is closed will automatically move to this column, including duplicates. My choice to assign to this column was reflective of this, applied manually to this issue as it had already been closed prior to the creation of the project board. I suspect this will not be the only issue which is closed as a result of being a duplicate, or needing to be moved elsewhere (Trac).

Would one of the following options help improve trackability?

- Move the favored issue #13550 into the "To do" column
   - Technically the issue was not part of the audit, however
- Rename the "Done" column to "Closed"
   - Better reflect that it's not necessarily done, simply that it's closed
- Create a separate column for issues closed by special circumstances (duplicates, moved)
- Invert the close; keep this one open, and close #13550 instead
   - Applied generally, this risks losing useful context if there was more useful conversation in the other issue

**Comment by rpkoller** [comment]
I would vote to move #13550 into the "to do" column. even though it is not part of the audit but it is original issue #15338 was a duplicate of.

**Comment by afercia** [comment]
I've moved #13550 into the "To do" lane.
